terraform/helm/aptos-node/values.yaml

```diff
@@ -14,6 +14,10 @@ numValidators: 1
 # -- Total number of fullnode groups to deploy
 numFullnodeGroups: 1
 
+multicluster:
+  enabled: false 
+  targetClusters: ["cluster1", "cluster2", "cluster3"]
```

Comment:
Please make this 0, 1 and 2.

Reply:
So, 0 is kind of special, it refers to the karmada master cluster in which you cannot schedule any other pods, so 1, 2, 3.